kuma/wiki/tests/test_views.py

```diff
@@ -562,13 +561,11 @@ def test_dont_render_previews_for_deferred_docs(self, mock_post):
 class DocumentSEOTests(UserTestCase, WikiTestCase):
     """Tests for the document seo logic"""
 
-    def test_get_seo_parent_doesnt_throw_404(self):
-        """bug 1190212"""
-        doc = document(save=True)
-        slug_dict = {'seo_root': 'Root/Does/Not/Exist'}
-        _get_seo_parent_title(doc, slug_dict, 'bn-BD')  # Should not raise Http404
-
-    def test_seo_title(self):
+    # NOTE(djf): In the past, we included the title of the "SEO Root"
+    # document in the title of every page, and this test tested for it.
+    # That feature has been removed now, and now this test verifies that
+    # the SEO root title does *not* appear in document titles.
+    def test_no_seo_title(self):
```

Comment:
For me, I think something like `test_no_seo_root_in_title` would be clearer, but this is fine too.